libcxx/include/deque

```diff
@@ -605,7 +605,8 @@ private:
 
 public:
```

Comment:
@changkhothuychung I think you need to update the synopsis in the header and also in the tests with the constexpr specifier.